apps/fluent-tester/src/FluentTester/testPages.win32.ts

```diff
@@ -38,6 +39,11 @@ export const tests: TestDescription[] = [
     component: JSAvatarTest,
     testPage: HOMEPAGE_AVATAR_BUTTON,
   },
+  {
+    name: 'Badge Test',
```

Comment:
Did we want to add this test page on other endpoints?

If so, please make sure that the test checks whether a platform supports SVGs before showing svg-based badges. Otherwise the test won't show on UWP.

Reply:
Yes, I think it should be also on other platforms